package.json

```diff
@@ -52,8 +52,9 @@
   },
   "scripts": {
     "clean": "hardhat clean && rm -rf dist cache src/types",
-    "build:dev": "hardhat compile && tsc --project ./tsconfig.json",
-    "build:publish": "hardhat compile && tsc --project ./tsconfig.publish.json",
+    "build": "hardhat compile && tsc",
+    "build:dev": "yarn build --project ./tsconfig.json",
```

Comment:
probably don't need this command anymore then, since it's equivalent to build

Totally see why we need `yarn build` here. The reason I didn't use that instead of `build:dev` is b.c. I'm certain people will use the wrong build command before publishing and then publish the wrong artifacts. Not sure how to avoid this

Reply:
In fact, this will indeed happen when people build + publish from the monorepo

Reply:
I guess now that I think about it more, the build command can just be what build:publish is right? We don't need to formally build with the dev tsconfig, that's just so the tests + editor will work right